src/nvim/window.c

```diff
@@ -4670,7 +4680,9 @@ int win_comp_pos(void)
   // Too often, but when we support anchoring floats to split windows,
   // this will be needed
   for (win_T *wp = lastwin; wp && wp->w_floating; wp = wp->w_prev) {
-    win_config_float(wp, wp->w_width, wp->w_height, wp->w_float_config);
+    wp->w_float_config.width = wp->w_width;
```

Comment:
this can also be deleted

Reply:
Deleting that will make this test fail: https://github.com/neovim/neovim/blob/7bdb2f15d8e03e9b022b029bd44ec2bb7917df95/test/functional/ui/float_spec.lua#L589-L617

Reply:
Let's keep it for now then. I will try to fix it later.